src/components/ProductDetailCard/ProductDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import { Product } from '../../utils/dataTypes/product';
+import ReviewsList from '../ReviewsList/ReviewsList';
+import PlusMinusButton from '../PlusMinusButton/PlusMinusButton';
+import { useCart } from '../../ShoppingCartContext';
+import './ProductDetailCard.css';
+
+type ProductDetailCardProps = {
+	product_id: Product['product_id'];
+	product_name: Product['product_name'];
+	product_author: Product['product_author'];
+	category: Product['category'];
+	product_description: Product['product_description'];
+	price: Product['price'];
+	product_image: Product['product_image'];
+};
+
+export function ProductDetailCard({
+	product_id,
+	product_name,
+	product_author,
+	category,
+	product_description,
+	price,
+	product_image,
+}: ProductDetailCardProps) {
+	const { handleRemoveFromCart, handleAddToCart } = useCart();
+
+	return (
+		<>
+			<h2> See the book in more detail!</h2>
```

Comment:
Not the end of the world but instead of this heading, perhaps this could be the name of the book, e.g.
```html
<h2>{product_name}</h2>
<p>{product_author}</p>
```